src/lua/log.lua

```diff
@@ -81,13 +90,17 @@ local compat_v16 = {
 }
 
 return setmetatable({
-    warn = say_closure(S_WARN);
-    info = say_closure(S_INFO);
-    debug = say_closure(S_DEBUG);
-    error = say_closure(S_ERROR);
-    rotate = log_rotate;
-    pid = log_pid;
-    level = log_level;
+    -- say for level functions
+    warn     = say_closure(S_WARN),
```

Comment:
I don't get it. They both don't require special handling of the last element, they're simply separators. We're using commas everywhere, so why don't we use it there?
Last element and comma is a better way not to allow error in the code (in case when you're adding another key-value to the last line)